src/converters/lintConfigs/rules/convertRules.test.ts

```diff
@@ -353,6 +353,45 @@ describe("convertRules", () => {
         ]).toContainEqual(converted);
     });
 
+    it("merges when a rule's ruleSeverity explicitly differs from its TSLint equivalent", () => {
+        // Arrange
+        const conversionResult: {
+            rules: { ruleName: string; ruleSeverity: ESLintRuleSeverity }[];
+        } = {
```

Comment:
Nit: the array type there is the `ConvertedRuleChanges` exported by `ruleConverter.ts`.

```suggestion
        const conversionResult: {
            rules: ConvertedRuleChanges[];
        } = {
```

Although, I think this would be a bit cleaner:

```ts
const rules: ConvertedRuleChanges[] = [
    {
        ruleName: "eslint-rule-a",
        ruleSeverity: "off",
    },
];

const { tslintRule, converters, mergers } = setupConversionEnvironment({
    ruleSeverity: "error",
    conversionResult: { rules },
    ruleToMerge: rules[0].ruleName,
});
```

What do you think?

Reply:
I'll incorporate those changes - agree that it looks cleaner! :)